integration/test_project_management.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for the firebase_admin.project_management module."""
+
+from firebase_admin import project_management
+
+
+TEST_APP_PACKAGE_NAME = 'com.firebase.adminsdk_python_integration_test'
+TEST_APP_DISPLAY_NAME_PREFIX = 'Created By Firebase AdminSDK Python Integration Testing'
+
+
+def _ensure_test_android_app_exists():
```

Comment:
I think this should be a module-scoped fixture. Otherwise we will keep executing it for each test case. See `integration/test_db.py` for an example.

Reply:
How do I set up the default app? I don't see anything in `test_db.py` that performs such set-up. I keep getting an error that says default app is not set up correctly.

Reply:
There's a session scoped fixture named `default_app` in `integration/conftest.py` which gets automatically invoked for all integration tests.

You need to pass `--cert` and `--apikey` parameters to the `pytest` command in order to trigger integration tests correctly. See the `CONTRIBUTING.md` file for details.